File: android/src/main/java/com/witsystem/top/flutterwitsystem/serialport/SerialPort.java
package com.witsystem.top.flutterwitsystem.serialport;

/**
 * 蓝牙串口通信
 */
public interface SerialPort {

    /**
     * 添加串口的回调
     */
    void addCall(SerialPortListen serialPortListen);


    /**
     * 发送串口数据
     */
    boolean sendData(String deviceId, String data);


    /**
     * 关闭串口
     */
    void closeSerialPort();


}

File: ios/Classes/serialport/SerialPort.swift
//
// Created by yyjjls on 2020/7/10.
//

import Foundation

/**
 * 蓝牙串口通信
 */
protocol SerialPort {

    /**
     * 添加串口的回调
     */
    func addCall(serialPortListen: SerialPortListen);


    /**
     * 发送串口数据
     */
    func sendData(deviceId: String, data: String)->Bool;


    /**
     * 关闭串口
     */
    func closeSerialPort();


}
